src/common/meta/src/ddl_manager.rs

```diff
@@ -261,6 +262,9 @@ async fn handle_truncate_table_task(
             table_info_value,
             table_route,
         )
+        .trace(tracing::info_span!(
+            "DdlManager::submit_truncate_table_task"
+        ))
```

Comment:
Is there any method that can generate this literal?

Reply:
There are indeed some ways to generate function paths, which require some additional calculations (otherwise the function paths will be ugly), but the generated function paths are not `const str`, which does not meet the parameter requirements of `info_span`. If you must do it, you can only use procedural macros, which will be very troublesome.